src/Avalonia.Controls.TreeDataGrid/FlatTreeDataGridSource.cs

```diff
@@ -96,6 +99,11 @@ bool ITreeDataGridSource.SortBy(IColumn? column, ListSortDirection direction)
             return false;
         }
 
+        public IEnumerable<object> GetModelChildren(object model)
```

Comment:
Should probably be an explicit implementation so it's hidden by default (as it's a no-op in this case)

```suggestion
        IEnumerable<object> ITreeDataGridSource.GetModelChildren(object model)
```